examples/plot_global_reject.py

```diff
@@ -72,17 +75,24 @@
 plt.tick_params(axis='x', which='both', bottom='off', top='off')
 plt.tick_params(axis='y', which='both', left='off', right='off')
 
-colors = ['#E24A33', '#348ABD', '#988ED5']
+colors = ['#E24A33', '#348ABD', '#988ED5', 'k']
 
 plt.plot(scaling * param_range, scaling * test_scores,
          'o-', markerfacecolor='w',
          color=colors[0], markeredgewidth=2, linewidth=2,
          markeredgecolor=colors[0], markersize=8, label='CV scores')
+plt.plot(scaling * param_range,
+         scaling * gp.predict(np.array(param_range)[:, None]),
```

Comment:
@agramfort what did you intend to do here? We don't have access to the GP through this API ... was your plan to expose it? If so, how?

Reply:
feel free to revert. I don't remember